client/src/components/GroupPage/GroupPage.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
You have this page built out under `components/` instead of `pages/` - can you move it to the appropriate directory?